Fix get_wheel_filename being out of date (#39965)
ray-project/ray: doc/source/ray-overview/installation.rst

.. note::

>   .. If changes are made to the list of wheel links below, ensure `get_wheel_filename()` in  `https://github.com/ray-project/ray/blob/master/python/ray/_private/utils.py` are up to date.

Reviewer: ```suggestion
  .. If you change the list of wheel links below, remember to update `get_wheel_filename()` in  `https://github.com/ray-project/ray/blob/master/python/ray/_private/utils.py`.
```
